src/main/java/com/baeldung/web/controller/UserController.java

```diff
@@ -2,14 +2,15 @@
 
 import java.util.Locale;
 
-import com.baeldung.security.ActiveUserStore;
-import com.baeldung.service.IUserService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.baeldung.security.ActiveUserStore;
+import com.baeldung.service.IUserService;
 
-@Controller
+@RestController
```

Comment:
is this change needed here, since the mapping in this controller don't have a ResponseBody annotation, and return views instead?

Reply:
Technically no, I've just changed it for the sake of consistency. Let me know if I should revert this  change. Thanks